### DISCUSSION TITLE
High cost: Container List and Create

### DISCUSSION THREAD

**Discussion by Segflow** [Azure/azure-storage-fuse#1309] [Q&A] [answered]
We are using this on azure, we saw that a lot of the cost comes from "LRS List and Create Container Operations" and from this components. 

Wandering if there is anything we can do to reduce this? Is there any option that can slow down how often we list containers?

**Comment by bhansalivikas** [comment, marked as answer]
We do not have any flow to create container from blobfuse ? 
Listing will happen whenever kernel wants to get the list of files/folders from a given directory. Other places where we execute a list call is when kernel asks for properties of a file/folder (in this case listing will be very specific to that particular file only).
Kernel me shoot lot of such calls if you have not enabled proper caching. Ensure you have attr-cache enabled in blobfuse and there are bunch of timeout parameters in libfuse section you need to provide values to those as well. Refer to our base or sample config to learn about these.
